src/Controller/interPageController.py

```diff
@@ -299,15 +319,35 @@ def show_patient(self, path):
         """
         Display patient data
         """
+        patient_dict_container = PatientDictContainer(
```

Comment:
This is the significant change (not just automated reformatting based on black and isort).  Create the PatientDictContainer